hosts_example.yml

```diff
@@ -158,13 +158,30 @@ all:
     #       topics: "test_topic"
 
     #### Configuring Role Based Access Control ####
```

Comment:
Just a question on vars/ docs. So there are variables you need to set for both internal mds AND external mds. and then there are vars that should only be set if its an external mds? Are there any vars that need to be set only for internal mds?

I think the external mds section should go at the bottom of this rbac block and say "If connecting to external mds, all above vars must be set with the addition of the mandatory ones below" or something

Reply:
Good question.  Based on the current documentation in this file, no.  You would need all the properties for the external MDS.  Moving external MDS lower makes sense.  Will update now.